backend/src/services/NewsService.ts

```
@@ -0,0 +1,122 @@
+      data: news,
+      currentPage: page,
+      totalPages: Math.ceil(totalNews / limit),
+      totalItems: totalNews,
+    };
+  } catch (error) {
+    console.error('Error fetching paginated news:', error);
+    return { data: [], currentPage: page, totalPages: 0, totalItems: 0 };
+  }
+};
+
+export const getSingleNews = async (id: String): Promise<Response<INews>> => {
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)